lib/core/realtime/hotelClerk.js

```diff
@@ -619,30 +569,46 @@ class HotelClerk {
           }, {});
       }
 
-      await this.kuzzle.pipe(
-        'core:hotelClerk:removeRoomForCustomer',
-        { requestContext, room: roomPayload });
-    }
+      await this.kuzzle.pipe('core:realtime:user:unsubscribe:after', {
+        requestContext,
```

Comment:
Why sending the whole `requestContext` if we need only the `connectionId` ? (that's the only information inside)

Reply:
Honestly? I'm not sure why. I had some logic behind it, and I was about to explain it here, but then it occured to me that it just doesn't hold water. Especially considering other events I wrote that take a connectionId, such as `core:realtime:user:remove` or `core:realtime:unsubscribe` :thinking: 

I'll change it :+1: